blockly-core/generators/javascript/functional_procedures.js

```diff
@@ -75,6 +75,14 @@ Blockly.JavaScript.functional_call = function() {
   return code;
 };
 
+Blockly.JavaScript.functional_pass = function() {
+  // Call a functional procedure with a return value.
```

Comment:
It does have an associated block. It's these black ones with the different notches. The comment should be updated, thx.
![image](https://cloud.githubusercontent.com/assets/1767466/6193647/4a5a81b4-b372-11e4-93b9-461d7856075e.png)